app/src/main/java/com/sun/spoonacular/ui/base/MainFragment.kt

```diff
@@ -45,10 +51,17 @@ class MainFragment : Fragment(), BottomNavigationView.OnNavigationItemSelectedLi
         hideKeyboard()
     }
 
-    override fun onPageScrollStateChanged(state: Int) {}
+    override fun onPageScrollStateChanged(state: Int) {
+    }
 
     private fun setUpViewPager() {
-        containerViewpager.adapter = MainMenuViewPagerAdapter(childFragmentManager)
+        if (arguments?.get(BUNDLE_LIST_RECIPE_KEY) != null) {
+            recipes = arguments?.get(BUNDLE_LIST_RECIPE_KEY) as ArrayList<Recipe>
+        }
```

Comment:
Thử viết như này xem có được k =)) 
```suggestion
            recipes = (arguments?.get(BUNDLE_LIST_RECIPE_KEY) as? ArrayList<Recipe>)?: ArrayList<Recipe>()
```